### PR TITLE
Es6 refactor in /lib folder

### PR DESCRIPTION
**Pull request by Orlandster98**
**What kind of change does this PR introduce?**
Es6 refactor in /lib folder

**Did you add tests for your changes?**
No

**If relevant, link to documentation update:**
N/A

**Summary**
The Refactoring makes the code more consistent to the ES6 standards. issue #4099

**Does this PR introduce a breaking change?**
No

**Other information**
Thanks for the review.

### CODE REVIEW

File: lib/JsonpMainTemplate.runtime.js [hunk 1]
```diff
@@ -3,49 +3,52 @@
 	Author Tobias Koppers @sokra
 */
 /*globals hotAddUpdateChunk parentHotUpdateCallback document XMLHttpRequest $require$ $hotChunkFilename$ $hotMainFilename$ */
+"use strict";
```
Comment (sokra) [inline lib/JsonpMainTemplate.runtime.js]: Please don't refactor this file. It should be in low level js as it's embedded into the runtime code

Comment (Orlandster98) [inline lib/JsonpMainTemplate.runtime.js]: Thanks for the feedback. I'm very new to the project and it helps me a lot! I will revert the commit and there's also another thing in the `ContextModuleFactory.js` I have to fix.

**Comment by Orlandster98** [comment]
@sokra Sorry, currently I'm struggling with the CI build error caused by `ContextModuleFactory.js`. It tells me "Block-scoped declarations (let, const, function, class) not yet supported outside strict mode". Do you have any idea why this happens? It's also failing without all the "const", "let" and "=>" .  Seems to be something with the class declaration.

**Comment by sokra** [comment]
> It tells me "Block-scoped declarations (let, const, function, class) not yet supported outside strict mode".

Don't touch the `lib/JsonpMainTemplate.runtime.js` file.

**Comment by sokra** [comment]
Thanks

**Pull request merged**